projects/storefrontapp-e2e-cypress/cypress/e2e/vendor/cdc/b2b/b2b-scenarios.core-e2e.cy.ts

```diff
@@ -4,20 +4,30 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { newAddress } from '../../../../helpers/address-book';
+import { editedAddress } from '../../../../helpers/address-book';
 import { loginUser } from '../../../../helpers/checkout-flow';
 import * as alerts from '../../../../helpers/global-message';
 import * as cdc from '../../../../helpers/vendor/cdc/cdc';
-import {
-  cdcB2BDelegateAdminUser,
-  waitForCmsComponentsToLoad,
-} from '../../../../helpers/vendor/cdc/cdc';
+import { waitForCmsComponentsToLoad } from '../../../../helpers/vendor/cdc/cdc';
+import * as b2bCheckout from '../../../../sample-data/b2b-checkout';
 
 describe('CDC B2B scenarios', () => {
+  before(() => {
+    cy.window().then((win) => win.sessionStorage.clear());
+    Cypress.env('BASE_SITE', b2bCheckout.POWERTOOLS_BASESITE);
+    Cypress.env('OCC_PREFIX_USER_ENDPOINT', b2bCheckout.USER_REQUEST_ENDPOINT);
+    Cypress.env(
+      'OCC_PREFIX_ORDER_ENDPOINT',
+      b2bCheckout.ORDER_REQUEST_ENDPOINT
+    );
+  });
+
```

Comment:
@SherwinVarghese would you mind explaining in the PR description the motivation of the changes done in e2e tests. I feel I'm lacking some context and you can help me understand it. 
e.g. Did those tests not pass before and now we're fixing them? Or now we add new test cases that we omitted before?
Anyway, I can see those e2e are still work in progress, so no need to answer now. Maybe it just clarifies itself, when all e2e change are finished.

Reply:
sure @Platonn,

We have indeed added a few e2e tests last time.
Since we are touching the update scenario, I believe we would need to test the scenario by adding 2 addresses, setting one as default and then deletion of both the addresses. This is to double check that we have not left any scenarios open with the code changes.